admin/acceptance.py

```diff
@@ -964,6 +964,16 @@ def postOptions(self):
         provider = self['provider'].lower()
         provider_config = self['config'].get(provider, {})
 
+        if self.get('flocker-version') is not None:
+            try:
+                parse_version(self['flocker-version'])
+            except UnparseableVersion:
+                raise UsageError(
+                    "Flocker version {!r} is not a valid version".format(
+                        self['flocker-version']
+                    )
+                )
+
```

Comment:
This a nice model. However, we don't do this anywhere else, so it causes a consistency problem.  We should probably adopt this model universally, so developers understand the idiom.